pulsar-broker/pom.xml

```diff
@@ -54,6 +54,18 @@
       <artifactId>netty-all</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>pulsar-common</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
```

Comment:
Why do we need to exclude checksum here?

Reply:
because bookkeeper-server-shaded jar include the circe-checksum when shading guava. so circe-checksum is effectively available in pulsar-broker when including `bookkeeper-server-shaded` jar. so excluding pulsar-common to make the behavior more deterministic. 

that said, we probably don't have to exclude it though. because in the classpath we have both shaded guava and non-shaded guava.